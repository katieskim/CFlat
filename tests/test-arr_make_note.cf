int main()
{
  note n;
  note m;
  note k;
  note[] notearr;

	n = ( /C-/ /4/ /q/ );
  m = ( /E/ /5/ /w/ );
  notearr = make(note, 6);

  notearr[0] = n;
  notearr[1] = m;
  k = notearr[1];
  (: notearr[2] = m; :)

  printn(k);
  return 0;
}
