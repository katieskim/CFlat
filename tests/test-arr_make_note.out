/E/ /5/ /w/ 